packages/thirdweb/src/auth/core/generate-login-payload.ts

```diff
@@ -35,7 +35,10 @@ export function generateLoginPayload(options: AuthOptions) {
       // use the user passed nonce generator or fall back to generationg uuid
       nonce: await (
         options.login?.nonce?.generate ||
-        (() => import("../../utils/uuid.js").then(({ uuid }) => uuid()))
+        (() =>
+          import("../../utils/uuid.js").then(({ randomBytes32 }) =>
```

Comment:
maybe would make sense to rename this file since it no longer has any uuid stuff in it?